src/components/BaseComponents/BaseHeader.tsx

```diff
@@ -36,10 +39,22 @@ const styles = (theme: Theme) =>
       float: 'right',
     },
     account: {
-      color: theme.palette.divider,
+      color: theme.palette.primary.main,
       fontSize: '30px',
       padding: 0,
     },
+    searchBar: {
+      position: 'absolute',
+      width: '100%',
+      padding: '20px',
+      backgroundColor: 'white',
+    },
+    leftTitle: {
+      float: 'left',
+      padding: '0px 25px',
+      fontSize: '30px',
```

Comment:
Instead of setting a custom font size here, condition the variant in the `Typography` instance. Something like this:
`<Typography ... variant={searchAction ? 'h1' : 'h2'}>{title}</Typography>`

I know the effect is the same, but we want to try to stick to the material standardized styles as much as we can.